### ISSUE TITLE
FR: Add missing support for other fiat currencies to support accessibility for other countries

### ISSUE THREAD

**Issue by tohrxyz** [sideswap-io/sideswapclient#32]
Hey, I really enjoy the user experience sideswap app provides and it brings a lot of value.

However as an european, it's not great for me to only see USD amounts of balance. It brings troubles and the need of external calculations. 

a solid wallet app should support multiple fiat currencies for display, like EUR.

Thanks for the ongoing effort in this project and I am looking forward to what comes next!

**Comment by sideswap-io** [comment]
Thank you for the suggestion. We've implemented the feature, and it will be available in the upcoming release.

**Comment by sideswap-io** [comment]
Feature is included in current release 1.4.8

**Comment by tohrxyz** [comment]
Thanks for adding this feature, it's great! @sideswap-io 

Closing this issue.
